kratos/utilities/delaunator_utilities.cpp

```diff
@@ -0,0 +1,218 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+// System includes
+
+// External includes
+#define REAL double
+
+#if !defined(KRATOS_TRIANGLE_EXTERNAL_H_INCLUDED)
+#define  KRATOS_TRIANGLE_EXTERNAL_H_INCLUDED
+#include "triangle.h"
+#endif
+
+#ifdef INCLUDE_DELAUNATORCPP
+#include "delaunator.hpp"
```

Comment:
here it is still

Reply:
It is not possible to define it as an external dependency, not mandatory?